EIPIP Meeting 25 Notes.md

```diff
@@ -0,0 +1,318 @@
+
+**James**  - There has been a little bit of a gap of what to do with the articles that Tim Beiko has written basically, and micah as well, like some of the more 'why we're doing something' and less of the spec kind of idea.
+
+**Micah**  - There's definitely a missing piece exactly as James just said where you've got the spec which is a technical specification. It's kind of motivational section but that's meant to be like a brief little thing just to get people in the mood. It's not meant to be an argumentative essay. It's not meant to be an opinion piece, it's the motivation section - like "hey here's some things that this would be useful for". Like here's a short list of uses or here's a few benefits we get when playing this. Meanwhile of course along with the EIP there's also the opinion pieces, there's the user stories, there's the essays, there's the analyses that are being done -  eip 1559 is a great example of this because we have a ton of this stuff and there's currently no place to put that right now. Tim just has a hack.md article or page that he's creating...
+
+**Brent** - Did we lose Micah?
+
+**Hudson** - Yes,we lost Micah. I think the summary of what Micah was saying... unless, Micah you back?
+
+**Micah** - Where did I cut off there?
+
+**Hudson** - You were talking about what was missing or the the fact that there were definitely some things that were missing and there were articles that were written that provided supplemental information that could need to go somewhere.
+
+**Micah** - Yeah that's basically it. We've got eip 1559, a great example of this. We've got tons of supplemental information and nowhere to put it and so we can link people to it of course when we have a channel and we have people coming in and talking to it but there's no place that people can know to go to read more about any supplemental information about an eip because we want to keep that out of the eip itself because the eip is a technical document. But we don't have somewhere else to send the person to say "hey if you ever want to know more, like opinion stuff on an eip, go here."
+
+**Hudson** - Ah okay 
+
+**Pooja** - So that is the thing that we are proposing here. I'm gonna share the link in the chat - this is the place i mean. On the ethereum cat header website we are trying to curate a list of resources for any eip under discussion. For example right now we have three or four eips;  1559, 2938, evm384 and all that stuff, so whatever resources other than the eip at magicians forum, like blogs published by you and others, we curate it and keep it there. The idea here is that we can make it publicly available with the help of ethereum.org. They are thinking in this direction and looking for some thoughts like if this is something good that we should be doing.
```

Comment:
```suggestion
**Pooja** - So that is the thing that we are proposing here. I'm gonna share the [link in the chat](https://github.com/ethereum-cat-herders/ethereum-cat-herders.github.io/blob/master/index.md#1559-fee-market-change). On the ethereum cat header website we are trying to curate a list of resources for any eip under discussion. For example right now we have three or four eips;  1559, 2938, evm384 and all that stuff, so whatever resources other than the eip at magicians forum, like blogs published by you and others, we curate it and keep it there. The idea here is that we can make it publicly available with the help of ethereum.org. They are thinking in this direction and looking for some thoughts like if this is something good that we should be doing.
```